heron/tools/tracker/src/python/routers/metrics.py

```diff
@@ -151,19 +152,19 @@ class TimelinePoint(BaseModel): # pylint: disable=too-few-public-methods
 
 class MetricsQueryResponse(BaseModel): # pylint: disable=too-few-public-methods
   """A metrics timeline over an interval."""
-  start_time: int = Field(..., alias="starttime")
-  end_time: int = Field(..., alias="endtime")
+  starttime: int = Field(..., alias="starttime")
+  endtime: int = Field(..., alias="endtime")
```

Comment:
Would be best to switch these to `str` as well for consistency as you have switched their similar definitions from `int` to `str` elsewhere? Or are these used in some sort of numeric calculation somewhere?

Reply:
I honestly don't remember the specifics, but I think the changes I made was due to an error I encountered. But it might have been self inflicted based on the various changes.  At the moment, I don't know the right answer, but I agree with your comment.